middleware-catalog-module/catalog-model/.project

```diff
@@ -5,13 +5,19 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
```

Comment:
This eclipse file must be in .ignore. Remove it and do git add, git commit, git push.